internal/procedures/users/upsert_test.go

```diff
@@ -382,42 +382,42 @@ func TestUpsertUser(test *testing.T) {
 			options: UpsertUsersOptions{
 				RawUsers: []*model.RawUserData{
 					&model.RawUserData{
-						Email: "admin@test.com",
```

Comment:
Can you not use "server-admin@test.com" instead of "owner@test.com"?

That applies to a lot of these tests.